src/libraries/System.Text.Json/src/System/Text/Json/ThrowHelper.Serialization.cs

```diff
@@ -513,20 +520,26 @@ public static void ThrowJsonException_MetadataDuplicateIdFound(string id)
             ThrowJsonException(SR.Format(SR.MetadataDuplicateIdFound, id));
         }
 
+        [DoesNotReturn]
+        public static void ThrowJsonException_MetadataDuplicateTypeProperty()
+        {
+            ThrowJsonException(SR.MetadataDuplicateTypeProperty);
```

Comment:
would be good to add extra info which property

Reply:
That would be reflected in the `JsonPath` property in the exception, so there's no need to explicitly specify it in the message.